res/skins/LateNight/waveform_and_mixer.xml

```diff
@@ -157,7 +157,7 @@
 					</Template>
 					<Template src="skin:waveform.xml">
 						<SetVariable name="channum">1</SetVariable>
-                        <SetVariable name="deck_color">#EECE33</SetVariable>
+                        <SetVariable name="deck_color">#E7C413</SetVariable>
```

Comment:
Sure, I can do that. It will have to be changed in decks_right.xml, decks_left.xml, deck_row_5.xml and deck.xml too though.

Should I change it in Deere too, to start making it more uniform across skins?